vignettes/advs.Rmd

```diff
@@ -143,12 +162,18 @@ the WHO standards, as no equivalent for \>2 years of age are provided by CDC.
 
 ### Reference Files for Weight by Length/Height
 
-WHO provides additional standards for weight by body length and weight
-by height, instead of age. For these we again combined the metadata for
-sex so that we have a single reference file by body length and a
-single reference file by height. We also give the values and units
-consistent variable names so that we can pass either metadata into the
-same function. We do this using the following code:
+WHO provides additional standards for weight by body length (recumbent length),
```

Comment:
Line 159 "The above example only shows the parameter BMI, but similar follows for the
other parameters like weight and height.". Should we mention here that there is no `Sigma` nor `P95` variables for Height and Weight ?

Reply:
added